tasks/CarryMyLuggage.tex

```diff
@@ -1,84 +1,82 @@
-\section{Carry My Luggage [Party Host]}
```

Comment:
I notice the theme tags have been dropped. Even though most are obvious, I think there should be a listing somewhere to be unambiguous. A placement with less emphasis on the this page makes sense to me

Reply:
Listing is at the start of Stage 1 and Stage 2 chapters